contrib/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# docker build -t schedviz .
+# docker run -d -p 127.0.0.1:7402:7402 schedviz
+
+FROM debian
+
+USER root
+
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y build-essential unzip git
+RUN apt-get install -y npm
```

Comment:
This appears to install an old version of node/npm, which prints warnings about the version of npm and node being incompatible. It works, but I wonder if it always will.

I think you need to install Node like [this](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions-enterprise-linux-fedora-and-snap-packages).

Reply:
Still looking into the Node version issue - yarn appears to want Node 9 or older, the apt process installs Node 10.
"npm WARN npl npm does not support Node.js v10.21.0"